app/models/spree/admin/tabs/root.rb

```diff
@@ -0,0 +1,25 @@
+module Spree
+  module Admin
+    module Tabs
+      class Root
+        attr_reader :items
+
+        def initialize
+          @items = []
+        end
+
+        def add(item)
```

Comment:
I think we should also have a method for inserting after/before another item (see the `insert_after` and `insert_before` methods in main menu). Some of the existing extensions inject their tabs e.g. closer to another tab that's logically close to it. I think we should allow them to do that as well, so that they don't have a reason to use deface anymore.